UnitTests/ScheduleTests/MonthsOnTheLastWeekDayTests.cs

```diff
@@ -0,0 +1,105 @@
+namespace FluentScheduler.Tests.UnitTests.ScheduleTests
+{
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using System;
+
+    [TestClass]
+    public class MonthsOnTheLastWeekDayTests
+    {
+        [TestMethod]
+        public void Should_Add_Specified_Months_To_Next_Run_Date_And_Select_Last_Day_In_That_Month()
```

Comment:
I know this is not a great deal but test name does not fit the library coding style.